net/leech/Makefile

```diff
@@ -0,0 +1,61 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=leech
+PKG_VERSION:=1.7
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://bitbucket.org/alekseyt/leech.git
+PKG_SOURCE_DATE:=2019-07-17
+PKG_SOURCE_VERSION:=2378e7348a8ac8ed23b6d29be81dc3a73080d920
+PKG_MIRROR_HASH:=04ca1d1f2cf614edf871360c1adbc758d9b85b67296af438449367c068ce5320
+
+PKG_MAINTAINER:=Leong Hui Wong <wong.leonghui@gmail.com>
+PKG_LICENSE:=blessing
+PKG_LICENSE_FILES:=COPYING
+
+PKG_INSTALL:=1
```

Comment:
This can probably be removed if Build/Install is to be empty.

Reply:
Ok, done.